TsRandomizer/Localizer.cs

```diff
@@ -46,5 +46,23 @@ public void OverrideKey(string key, string value)
 			{
 			}
 		}
+
+		public void ResetStrings()
```

Comment:
plz use tabs instead of spaces, i added an .editorconfig that should have prevented this